### ISSUE TITLE
Error with overwriting file used to render inline citations.

### ISSUE THREAD

**Issue by gnramos** [francisco-perez-sorrosal/mdbook-bib#43]
I want to adjust reference style for my mdbook, But I'm having a little trouble with author names. I tried the default approach by downloading [references.hbs](https://github.com/francisco-perez-sorrosal/mdbook-bib/blob/master/src/render/references.hbs), adding it to my `src ` folder and then adding `hb-tpl = "references.hbs"` to the TOML file as per the manual's instructions. Compilation is fine, but the resulting bibliography is missing the list of authors.

In the HTML I get:
```
<summary class=citation>
<a id="Knuth1974">[Knuth1974]</a> -  - <a href="http://portal.acm.org/citation.cfm?doid&#x3D;361604.361612" target="_blank"><cite>Computer programming as an art</cite></a>. - 1974. -
<button onclick="copyToClipboard('\{\{ #cite Knuth1974 \}\}')">Copy citation_key</button>
</summary>
```

instead of the expected (and default behavior if no  `hb-tpl` is set in the TOML):
```
<summary class=citation>
<a id="Knuth1974">[Knuth1974]</a> - Knuth, Donald E. - <a href="http://portal.acm.org/citation.cfm?doid&#x3D;361604.361612" target="_blank"><cite>Computer programming as an art</cite></a>. - 1974. -
<button onclick="copyToClipboard('\{\{ #cite Knuth1974 \}\}')">Copy citation_key</button>
</summary>
```

**Comment by mlange-42** [comment]
@gnramos Thanks for reporting! However, I am not able to reproduce this.

Could you provide a minimal reproducing setup, or a link to the affected repository in case it is public?

Thanks!

**Comment by gnramos** [comment]
Hello @mlange-42 . Thanks for addressing this. I changed my project along the way, so I cannot reproduce this issue as I found it anymore. I changed it to include the `hbs` I just downloaded from this repo and it worked as expected. I also tried to make a mwe from what I described/remembered, but failed to reproduce this. Thus, I am closing this issue. 
ps: Thanks for providing this functionality to mdbook.
